test/JDBC/input/BABEL-SQUARE-vu-prepare.sql

```diff
@@ -0,0 +1,6 @@
+-- Test select from table
+CREATE TABLE cubes(id int, side_length float(24));
```

Comment:
the naming convention is filename_xxx, so it should be BABEL-SQUARE-vu-prepare_cubes, applied to all objects created as part of this change